cerberus/validator.py

```diff
@@ -428,6 +435,19 @@ def allow_unknown(self, value):
             DefinitionSchema(self, {'allow_unknown': value})
         self._config['allow_unknown'] = value
 
+    @property
+    def require_all(self):
+        """ If ``True`` known fields that are defined in the schema will
+            be required.
+            Type: :class:`bool` """
+        return self._config.get('require_all', False)
+
+    @require_all.setter
+    def require_all(self, value):
+        if not (self.is_child or isinstance(value, (bool, DefinitionSchema))):
```

Comment:
you musn't copy that from `allow_unknown` where it has the purpose to validate the provided value.

Reply:
`require_all` should not have setter? I'm not familiar with internals, sorry.

Reply:
it should have a setter. but the check in line 447-448 is specific to `allow_unknown`. just assign the value like `purge_readonly` for example.